src/01-functions.js

```diff
@@ -112,7 +123,14 @@ function averageThree() {}
  *  compareTypes("left", 10);
  *  //> false
  */
-function compareTypes() {}
+function compareTypes(a,b) {
+  if (typeof a === typeof b) {
+    return true
+  } else {
+    return false
+  }
+
```

Comment:
Watch this trailing extra line of space - if whitespace doesn't make our code easier to read we should lose it